### PR TITLE
Map move, power ups type send to client fix, player knock from left b…

### PR DESCRIPTION
**Pull request by lukasziukas**
Map move, PowerUps type send fix

### CODE REVIEW

File: Core/NoOpRunner.Core/NoOpRunner.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public class NoOpRunner : ISubject
     {
         public bool IsGameStarted { get; set; } = false;
 
-        public event EventHandler OnLoopFired;
+        // public event EventHandler OnLoopFired;//Un
```
Comment (Kodnot) [inline Core/NoOpRunner.Core/NoOpRunner.cs]: ?

File: Core/NoOpRunner.Core/NoOpRunner.cs [hunk 2]
```diff
@@ -56,6 +56,26 @@ public NoOpRunner(IConnectionManager connectionManager)
...
+            PlatformsContainer.MoveWithMap();
+            
+            await connectionManager.SendMessageToClient(new MessageDto()
+            {
+                MessageType = MessageType.PlatformsUpdate,
+                Payload = PlatformsContainer.GenerateSequel()
+            });
+        }
+        /// <summary>
+        /// Eh??? Useless?? ORAORAORAORAORAORA
```
Comment (Kodnot) [inline Core/NoOpRunner.Core/NoOpRunner.cs]: Yes, unused right now, the powerup generation logic won't be here anyways, can remove

**Review by Kodnot — CHANGES_REQUESTED** [review]
Also, power ups are still not quite synced properly - they don't go off screen for P2 and just remain at the bottom left corner

File: Core/NoOpRunner.Core/NoOpRunner.cs (hunk 1 above)
Comment (lukasziukas) [reply Core/NoOpRunner.Core/NoOpRunner.cs]: Unused from the beginning of project, checked if fails to run after commenting

**Review by Kodnot — APPROVED** [review]

**Pull request merged**